hollow/src/main/java/com/netflix/hollow/api/codegen/indexes/HollowUniqueKeyIndexGenerator.java

```diff
@@ -109,6 +116,7 @@ protected void genPublicAPIs(StringBuilder builder) {
     }
 
     protected void genFindMatchAPI(StringBuilder builder) {
+        builder.append("    @Override\n");
```

Comment:
It should have an explicit condition. It worked that way in effect because the `PrimaryKeyIndexGenerator` overrode `genFindMatchAPI`, but it's more correct and easier to understand with your suggestion.